docs/integrator-guide.adoc

```diff
@@ -1539,7 +1588,7 @@ To make AsciidoctorJ use our processor it also has to be registered at the `Java
 <4> Test via the Jsoup HTML parsing library that our meta tag was correctly added to the resulting document.
 
 [[ExtensionSPI]]
-=== Extension SPI
+===Automatically loading extensions
```

Comment:
Can you please add a blank between the `===` and `Automatically`?
Asciidoctor won't recognise it as a heading like this.